docs/News.md

```diff
@@ -1,3 +1,10 @@
+## August 14, 2024
+
+*   **New feature**: There is a new interactive 3D visualization software tool integrated with cBioPortal, for users running Windows. Once installed, a button will be available on all group comparison data tables, which will load it in the software with an interactive 3D Landscape with data shaping tools, including integrated pathways. Read more on [AVM for cBioPortal](https://bit.ly/avm-cbioportal). 
+    * **Customization Note**: This button is not configured by default in private instances. Please see [Customization](deployment/customization/download_custom_buttons-Reference.md).
```

Comment:
used relative cross-link (per other PR) so it works in the github repo browser as well

Reply:
@pappde Thanks for the draft. I suggest we simplify the text as below. 

*   **New feature** (_Windows user only_): The Group Comparison results can now be visualized in 3D with AVM, an interactive 3D visualization software tool. Read more on [AVM for cBioPortal](https://bit.ly/avm-cbioportal). 
    * **Note for private instances**: To enable this feature in private cBioPortal instances, please follow [this instruction](deployment/customization/download_custom_buttons-Reference.md).

Adding @tmazor @inodb @schultzn for comments. Note that the feature is already out and it's also visible here (https://www.cbioportal.org/visualize).

Reply:
@jjgao Updated with new phrasing. See the new diff. Also entering in this comment to reflect the latest commit:

*   **New feature** (_Windows users only_): The Group Comparison results can now be visualized in 3D with AVM, an interactive 3D visualization software tool with data shaping functions and integrated pathways. Read more on [AVM for cBioPortal](https://bit.ly/avm-cbioportal). 
    * **Note for private instances**: To enable this feature in private cBioPortal instances, please see [this instruction](https://docs.cbioportal.org/deployment/customization/application.properties-reference#add-custom-buttons-to-data-tables).

I did include the "with .... pathways" feature note since "interactive 3D visualization" alone may imply a simple static representation of the table data, which isn't as useful.  

Also Note: the new feature allows data-driven definition of buttons. That's why the original text was titled "Customization" but I agree we really want to keep it simple here. So I changed the link to the application.properties reference document (which does actually link to the Customization document that was previously linked).